editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-spy-wrapper-third-party.spec.browser2.tsx

```diff
@@ -146,99 +145,57 @@ xdescribe('Spy Wrapper Tests For React Three Fiber', () => {
     // currentWindow.openDevTools()
     // await wait(20000)
 
-    const { getEditorState } = await createTestProject()
+    const { getEditorState } = await renderTestProject()
+    // React Three Fiber seems to have some second pass render that appears to run
+    // after the regular React render and this appears to give it a chance to be triggered.
+    await wait(100)
     const spiedMetadata = getEditorState().editor.spyMetadata
     const sanitizedSpyData = simplifiedMetadataMap(spiedMetadata)
     matchInlineSnapshotBrowser(
       sanitizedSpyData,
       `
       Object {
         "storyboard": Object {
-          "children": Array [
-            "storyboard/scene-1",
-            "storyboard/scene-2",
-          ],
           "name": "Storyboard",
-          "rootElements": Array [],
         },
         "storyboard/scene-1": Object {
-          "children": Array [
-            "storyboard/scene-1/canvas-app",
-          ],
           "name": "Scene",
-          "rootElements": Array [],
         },
         "storyboard/scene-1/canvas-app": Object {
-          "children": Array [],
           "name": "CanvasApp",
-          "rootElements": Array [],
         },
         "storyboard/scene-1/canvas-app:canvas-app-div": Object {
-          "children": Array [
-            "storyboard/scene-1/canvas-app:canvas-app-div/test-mesh",
-            "storyboard/scene-1/canvas-app:canvas-app-div/test-ambientLight",
-            "storyboard/scene-1/canvas-app:canvas-app-div/test-directionalLight",
-            "storyboard/scene-1/canvas-app:canvas-app-div/test-pointLight",
-          ],
           "name": "Canvas",
-          "rootElements": Array [],
         },
         "storyboard/scene-1/canvas-app:canvas-app-div/test-ambientLight": Object {
-          "children": Array [],
           "name": "ambientLight",
-          "rootElements": Array [],
         },
         "storyboard/scene-1/canvas-app:canvas-app-div/test-directionalLight": Object {
-          "children": Array [],
           "name": "directionalLight",
-          "rootElements": Array [],
         },
         "storyboard/scene-1/canvas-app:canvas-app-div/test-mesh": Object {
-          "children": Array [
```

Comment:
what's with these changes? did we remove the children and rootElements fields since this test was disabled?

Reply:
`simplifiedMetadataMap` strips them from the output.